docs/scos/dev/guidelines/nfr-guidelines.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Project Development Non-Functional Requirement guidelines
+description: Guidelines for defining and implementing Non-Functional Requirements (NFRs) to ensure the technical success of a project.
+last_updated: 
+template: concept-topic-template
+related:
+- title: Operatable feature guidelines
+  link: docs/scos/dev/guidelines/operatable-feature-guidelines.html
+- title: Process documentation guidelines
+  link: docs/scos/dev/guidelines/process-documentation-guidelines.html
+- title: Monitorable process guidelines
+  link: docs/scos/dev/guidelines/monitorable-process-guidelines.html
+- title: Architecture performance guidelines
+  link: docs/scos/dev/guidelines/performance-guidelines/architecture-performance-guidelines.html
+---
+
+Non-Functional Requirements (NFRs) are one of the core tools for architects and developers to describe how the system under development supposed to work from technical perspective.
+Defining NFRs can be as important as defining functional requirements, which are usually done by business people (i.e. Product Owners). Missing out on them usually result in
+too late realization of certain unintended behavior that costs a lot of resources/money for the projects to deal with.
+
+## NFR guidelines
```

Comment:
Reading the chapter, it gave me the impression that we are describing the regular NFRs (that appear during feature development). These 4 pages are NOT describing how to design features, but prepare the minimum requirements for project developers in order to ensure that the project can be operated & deployed on the infrastructure. These NFRs are not related directly to feature development (or POs or end-users).